samples/postman/README.md

```diff
@@ -5,33 +5,33 @@
 3. Select the **Variables** tab.
 4. Configure the 4 variables shown below by pasting them into the **Initial Value** and **Current Value** columns in Postman:
-    - `aud`: The URL that will recieve your token. 
+    - `aud`: The URL that will receive your token. 
         - To find this value, check the CCG documentation for the VA API you are working with.
         - Example (from Patient Health API): `https://deptva-eval.okta.com/oauth2/aus8nm1q0f7VQ0a482p7/v1/token`
     - `token_endpoint`: The URL you will request a token from.
         - To find this value, check the CCG documentation for the VA API you are working with.
         - Example (from Patient Health API): `https://sandbox-api.va.gov/oauth2/health/system/v1/token`
-    - `clientId`: The Client ID you recieved when you signed up for sandbox access.
-        - For information on signing up for sandbox access, check the documentation for the API you are working with in the [VA developer portal](https://developer.va.gov/explore).
+    - `clientId`: The Client ID you received by email when you signed up for sandbox access.
+        - For information on signing up for sandbox access, check the documentation for the API you are working with.
     - `privatePem`: The private RSA key you generated when signing up for sandbox access, in PEM format.
         - Your private RSA key should begin with `-----BEGIN RSA PRIVATE KEY-----` and end with `-----END RSA PRIVATE KEY-----`.
-        - For information on generating your RSA keys, check the CCG documentation for the API you are working with in the [VA developer portal](https://developer.va.gov/explore).
+        - For information on generating RSA keys, check the CCG documentation for the API you are working with.
 5. Do **not** edit the `client_assertion` and `import_do_not_edit` values. These will be pre-populated once the previous values are defined. 
-6. Now you're ready to request a token. Select **POST Client Credentials Example**, under **Lighthouse OAuth Token** in the left column of the Postman window.
-    - You should see that the HTTP URL is the `{token_endpoint}` collection variable that you configured in step `4`, above.
+6. Select **POST Client Credentials Example**, located under **Lighthouse OAuth Token** in the left column of the Postman window.
+    - You should see that the HTTP URL is the `{token_endpoint}` collection variable that you configured in step `4`.
 7. Select the **Body** tab.
 8. Configure the `scope` and `launch` values as follows:
-    - `scope`: The default value, "launch," is only appropriate for some VA APIs. Check the CCG documentation for the API you are working with in the [VA developer portal](https://developer.va.gov/explore) for the scope(s) that may be included in your token request. 
+    - `scope`: The default value, "launch," is only appropriate for some VA APIs. Check the CCG documentation for the API you are working with for the scope(s) that may be included in your token request. 
```

Comment:
Suggest removing the comma from "launch," so the variable name is more accurate.

Reply:
@mwise-va I just fixed that here and in the Postman collection itself.